app/routes/post_login.js

```diff
@@ -15,10 +15,6 @@ export default Route.extend({
     this.set("isBookAppointment", params.queryParams.bookAppointment);
   },
 
-  model() {
-    return this.get("preloadService").preloadData();
-  },
```

Comment:
Why has this been removed ? `model` is required. right ? Have you tested this thoroughly on local before creating the PR ?

Reply:
model is not used for post-login that's why I removed it